priv/test_repo/migrations/1_migrate_all.exs

```diff
@@ -6,8 +6,28 @@ defmodule ExMachina.TestRepo.Migrations.MigrateAll do
       add(:name, :string)
       add(:admin, :boolean)
       add(:net_worth, :decimal)
+      add(:db_value, :string)
     end
 
+    execute(~S"""
+    CREATE FUNCTION set_db_value()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+    BEGIN
+      NEW.db_value := 'made in db';
+      RETURN NEW;
+    END;
+    $$;
+    """)
+
+    execute(~S"""
+    CREATE TRIGGER gen_db_value
+    BEFORE INSERT ON users
+    FOR EACH ROW
+    EXECUTE FUNCTION set_db_value();
```

Comment:
Created this function and trigger so that we could properly test `returning: true`. Maybe it's testing too much? But it seemed like a good test that we're indeed passing the values to `Repo.insert!/2` as it expects them.